src/dotnet/APIView/APIViewWeb/Client/package.json

```diff
@@ -26,6 +26,7 @@
     "css-loader": "^3.2.0",
     "eslint": "^5.16.0",
     "eslint-plugin-vue": "^5.0.0",
+    "mark.js": "^8.11.1",
```

Comment:
Can we bundle this into the main JS in webpack?

Reply:
Removed